src/sentry/conf/server.py

```diff
@@ -1203,6 +1203,7 @@ def create_partitioned_queues(name):
 # Snuba configuration
 SENTRY_SNUBA = os.environ.get("SNUBA", "http://127.0.0.1:1218")
 SENTRY_SNUBA_TIMEOUT = 30
+SENTRY_SNUBA_CACHE_TTL_SECONDS = 10
```

Comment:
Just to check, there's no specific reason to make this 10s right? Could we set it to 60 so it matches the smallest allowed window in alerts?

Reply:
@JTCunning asked for 10 seconds initially, I don't mind increasing it. Especially since snuba is already caching, so there should be minimal product impact. Thoughts @JTCunning?

Reply:
60s is perfectly appropriate as long as we're keying on both `start` and `end` timestamps in their full accuracy.